contracts/stake/src/wasm.rs

```diff
@@ -162,7 +162,7 @@ impl StakeContract {
         let call = Transaction::from_canon(&call);
         let transfer = rusk_abi::transfer_contract();
 
-        dusk_abi::transact_raw(self, &transfer, &call)
+        dusk_abi::transact_raw(self, &transfer, &call, dusk_abi::gas_left())
```

Comment:
We shouldn't pass the `gas_left`. For the time being we can pass `0`, that scales to the default percentage of the gas to icc. In the future, we should pass the exact number we expect to use (but we need to calculate it first).

Same comment applies to the rest of the code with similar approach.